modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/BusyExecutor.java

```diff
@@ -35,52 +39,65 @@
     private final String name;
 
     /** Active flag (used to skip commands in inactive cluster.) */
-    private volatile boolean active;
+    private final AtomicBoolean active = new AtomicBoolean(false);
```

Comment:
```suggestion
    private volatile boolean active;
```

Reply:
Write access to active is already synchronized.

Reply:
Good point, thanks.